### DISCUSSION TITLE
JetBrains Rider Uno Platform Issue

### DISCUSSION THREAD

**Discussion by xauuzd** [unoplatform/uno#13042] [Q&A]
I am trying to create an Uno Platform App project with Rider. Everything works, except Android.

![image](https://github.com/unoplatform/uno/assets/84229419/8b4184a6-5104-49b0-a8f3-1611ace1fb27)

![image](https://github.com/unoplatform/uno/assets/84229419/c0c1ee7b-9db0-4a45-b36b-f6211031b545)

I tried to get help in the UWP Community (The discord server for Uno Platform), and I was told to escalate the issue to the Uno Platform GitHub. I am able to build the UWP (Windows) Project, but not Android.

By the way, I have not done anything manual to cause the errors that I get. When I create an Uno project from Rider, I get those errors automatically. But on Visual Studio, there's no issue.

Why?

**Comment by jeromelaban** [comment]
Thanks for the report! By the looks of the project, it seems like you're using an older template (the `.Droid` extension). Could you tell how you created your project and which version of visual studio you used to create it?

Make sure to run [uno-check](https://platform.uno/docs/articles/external/uno.check/doc/using-uno-check.html) to install the latest [dotnet new templates](https://platform.uno/docs/articles/get-started-dotnet-new.html?tabs=net7%2Cwindows).

Let us know how it goes?
